datamap/mv/plan/src/main/scala/org/apache/carbondata/mv/plans/modular/ModularRelation.scala

```diff
@@ -86,10 +87,28 @@ object HarmonizedRelation {
       Select(_, _, _, _, _, dim :: Nil, NoFlags, Nil, Nil, _),
       NoFlags,
       Nil, _) if (dim.isInstanceOf[ModularRelation]) =>
-        if (g.outputList
-          .forall(col => col.isInstanceOf[AttributeReference] ||
-                         (col.isInstanceOf[Alias] &&
-                          col.asInstanceOf[Alias].child.isInstanceOf[AttributeReference]))) {
+        if (g.outputList.forall(col => {
```

Comment:
better to move the logic out of if condition. and have it better formatted

Reply:
ok